test/cpp/jit/test_schema_matching.cpp

```diff
@@ -0,0 +1,96 @@
+#pragma once
+
+#include <ATen/test/test_assert.h>
+#include <torch/csrc/jit/ir.h>
+#include <torch/csrc/jit/irparser.h>
+#include <torch/csrc/jit/passes/constant_pooling.h>
+#include <torch/csrc/jit/passes/constant_propagation.h>
+#include <torch/csrc/jit/testing/file_check.h>
+#include <torch/jit.h>
+#include "test/cpp/jit/test_base.h"
+#include "torch/csrc/jit/custom_operator.h"
+
+#include <sstream>
+#include <string>
+
+namespace torch {
+namespace jit {
+
+void testSchemaMatching() {
+  auto run = [](std::shared_ptr<script::CompilationUnit> cu,
+                const std::string& name,
+                std::vector<IValue>& stack) {
+    auto graph = cu->get_function(name).graph();
+    Code code(graph);
+    InterpreterState interp(code);
+    interp.run(stack);
+    return stack;
+  };
+  {
+    RegisterOperators reg({
+        Operator(
+            "aten::test_vartype(t[] a, t b) -> (t)",
```

Comment:
use the testing namespace. Otherwise someone can write `torch.test_vartype` in the C++ frontend and it will work.

Reply:
Will use the testing namespace - however this op only gets defined when the test is run right ? i don't think `torch.test_vartype` would be defined

Reply:
as it is written, yes it will only appear here.